packages/foam-vscode/src/test/run-tests.ts

```diff
@@ -35,23 +33,17 @@ async function main() {
       // The path to the extension test script
       // Passed to --extensionTestsPath
       const extensionTestsPath = path.join(__dirname, 'suite');
-      const tmpWorkspaceDir = fs.mkdtempSync(path.join(os.tmpdir(), 'foam-'));
 
       // Download VS Code, unzip it and run the integration test
       await runTests({
         extensionDevelopmentPath,
         extensionTestsPath,
         launchArgs: [
-          tmpWorkspaceDir,
+          path.join(extensionDevelopmentPath, '.test-workspace'),
```

Comment:
The `.test-workspace` directory is part of the repo (that's why we need the `.keep` file) but its contents are ignored by git. This avoids having to create the directory in each entry point (launcher and suite.ts).